gms/impl/src/main/resources/DataPlatformInfo.json

```diff
@@ -2,86 +2,121 @@
   "urn:li:dataPlatform:adlsGen1": {
     "datasetNameDelimiter": "/",
     "name": "adlsGen1",
+    "displayName": "Azure Data Lake (Gen 1)",
     "type": "FILE_SYSTEM"
   },
   "urn:li:dataPlatform:adlsGen2": {
     "datasetNameDelimiter": "/",
     "name": "adlsGen2",
+    "displayName": "Azure Data Lake (Gen 2)",
     "type": "FILE_SYSTEM"
   },
   "urn:li:dataPlatform:ambry": {
     "datasetNameDelimiter": ".",
     "name": "ambry",
+    "displayName": "Ambry",
     "type": "OBJECT_STORE"
   },
   "urn:li:dataPlatform:couchbase": {
     "datasetNameDelimiter": ".",
     "name": "couchbase",
+    "displayName": "Couchbase",
     "type": "KEY_VALUE_STORE"
   },
   "urn:li:dataPlatform:external": {
     "datasetNameDelimiter": ".",
     "name": "external",
+    "displayName": "External Source",
     "type": "OTHERS"
   },
   "urn:li:dataPlatform:hdfs": {
     "datasetNameDelimiter": "/",
     "name": "hdfs",
+    "displayName": "HDFS",
     "type": "FILE_SYSTEM"
   },
   "urn:li:dataPlatform:hive": {
     "datasetNameDelimiter": ".",
     "name": "hive",
+    "displayName": "Hive",
     "type": "FILE_SYSTEM"
   },
   "urn:li:dataPlatform:kafka": {
     "datasetNameDelimiter": ".",
     "name": "kafka",
+    "displayName": "Kafka",
     "type": "MESSAGE_BROKER"
   },
   "urn:li:dataPlatform:kusto": {
     "datasetNameDelimiter": ".",
     "name": "kusto",
+    "displayName": "Kusto",
     "type": "OLAP_DATASTORE"
   },
   "urn:li:dataPlatform:mongo": {
     "datasetNameDelimiter": ".",
     "name": "mongo",
+    "displayName": "MongoDB",
     "type": "KEY_VALUE_STORE"
   },
   "urn:li:dataPlatform:mysql": {
     "datasetNameDelimiter": ".",
     "name": "mysql",
+    "displayName": "MySQL",
     "type": "RELATIONAL_DB"
   },
   "urn:li:dataPlatform:oracle": {
     "datasetNameDelimiter": ".",
     "name": "oracle",
+    "displayName": "Oracle",
     "type": "RELATIONAL_DB"
   },
   "urn:li:dataPlatform:pinot": {
     "datasetNameDelimiter": ".",
     "name": "pinot",
+    "displayName": "Pinot",
     "type": "OLAP_DATASTORE"
   },
   "urn:li:dataPlatform:postgres": {
     "datasetNameDelimiter": ".",
     "name": "postgres",
+    "displayName": "PostgreSQL",
     "type": "RELATIONAL_DB"
   },
   "urn:li:dataPlatform:presto": {
     "datasetNameDelimiter": ".",
     "name": "presto",
+    "displayName": "Presto",
     "type": "QUERY_ENGINE"
   },
   "urn:li:dataPlatform:teradata": {
     "datasetNameDelimiter": ".",
     "name": "teradata",
+    "displayName": "Teradata",
     "type": "RELATIONAL_DB"
   },
   "urn:li:dataPlatform:voldemort": {
     "datasetNameDelimiter": ".",
     "name": "voldemort",
+    "displayName": "Voldemort",
     "type": "KEY_VALUE_STORE"
+  },
+  "urn:li:dataPlatform:snowflake": {
+    "datasetNameDelimiter": ".",
+    "name": "snowflake",
+    "displayName": "Snowflake",
+    "type": "OLAP_DATASTORE"
```

Comment:
RELATIONAL_DB for Snowflake (align with Teradata)

Reply:
hmm... okay... technically idk

Reply:
should we have a warehouse type?